interface/package.json

```diff
@@ -14,7 +14,9 @@
   "engines": {
     "node": ">=4.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "hclusterjs": "^1.3.0"
```

Comment:
There is probably no any other benefits except the consistency. I am okay with hosting this js library in `node_modules/`.